src/assignments/benson/lesson_3_answers.clj

```diff
@@ -0,0 +1,128 @@
+(ns assignments.benson.lesson-3-answers)
+
+(declare flush? straight? filter-matching-pairs)
+
+(defn get-suit
```

Comment:
In clojure, we basically never write this sort of "lookup via a case statement" type function, we just use maps.

e.g.
(def suits {\s :spade, \h :heart ...etc})